pandas/tests/frame/conftest.py

```diff
@@ -127,6 +127,14 @@ def timezone_frame():
     return df
 
 
+@pytest.fixture
+def datetime_series():
+    """
```

Comment:
this fixture itself is not very useful, however, possibly a fixture that is a float_frame but uses this an index might be

Reply:
Yea I would agree with this comment. It also seems strange to put a fixture in this conftest which returns a series, as that could either create duplication or friction with the conftest for the series directory.

Any way to do what @jreback suggests above here?

Reply:
> > @h-vetinari: @jreback Can't do 3 things at the same time reasonably.
You're already requesting to do fixturization and renaming at the same time, let's keep the reorg for later. Also, case in point about the necessity of (some of) the fixtures re-added here: #25635

> @jreback: +1 on reorg of fixtures later

@WillAyd, of course both of you are right. But the quasi-fixtures from `tests.frame.common.TestData` that are supposed to be translated *already* have this (as `self.ts1`). I guess I should have pushed back harder against #24885, because these were already in `tests.frame.conftest` (since #22236) precisely for this translation. There's a limit to how many balls any one PR should have up in the air, and fixturization plus renaming is enough IMO.